src/include/ipc/dai.h

```diff
@@ -71,7 +71,8 @@ struct sof_ipc_dai_config {
 
 	/* physical protocol and clocking */
 	uint16_t format;	/**< SOF_DAI_FMT_ */
-	uint16_t reserved16;	/**< alignment */
+	uint8_t group_id;	/**< DAI group ID */
```

Comment:
Need a comment here saying that group ID 0 is reserved and means that the DAI doesnt belong to any group